README.md

```diff
@@ -31,6 +31,11 @@ Webhooks require a public URL that Stripe will ping to notify the monitor of new
 
 If you have a [__Basic__](https://ngrok.com/pricing) ngrok subscription, you can specify a custom subdomain that will stay reserved for your account.
 
+#### Optional: Verify Webhooks are signed
+[Stripe can optionally sign Webhook events to your endpoints](https://stripe.com/docs/webhooks/signatures). If you'd like the Webhook Monitor to verify these for you, simply modify the `signingSecret` value in `config.js` with the endpoint-specific signing secret key found at the bottom of the Webhook details page.
```

Comment:
Is it truer to say that these are always signed, and you can optionally _verify_ them?

Reply:
How about:
> Stripe can optionally [sign webhook events](https://stripe.com/docs/webhooks/signatures) so you can verify them on endpoints that receive webhooks. You can find the signing secret for each webhook endpoint [in the Dashboard](https://dashboard.stripe.com/webhooks) in the  _Signing secret_ section. To configure the Webhook Monitor to verify webhook signatures, specify the `signingSecret` in `config.js` using this secret key.
  @willock-stripe
Can we maybe include a screenshot?

Reply:
In a world where we provision this on the user's behalf, I don't think it's necessary to call this out at all anymore.